tsp-output/@azure-tools/typespec-csharp/src/Generated/Internal/ClientPipelineExtensions.cs

```diff
@@ -1,49 +1,37 @@
 // <auto-generated/>
 
-#nullable disable
+#nullable enable
```

Comment:
This is important for a "ClientModel" type so we can do proper validation on nulls.

Reply:
well the generated code will always have "nullable disable", therefore we cannot do this unless we change all of them in generated code (including models clients) and all other dependencies (such as resourcemanager)

Reply:
@ArcturusZhang, why is generated code not able to change `#nullable disable` to `#nullable enable` for select emitted files?

Reply:
We can change. But I believe we really do not have to change it now.
The generator does not support generating nullable context yet.

Reply:
@ArcturusZhang:  I think the question that needs to be answered here is why are we looking at this as "all or nothing?"    I would think that it should be possible to emit nullable in the scope of the unbranded writer without impacting DPG/MGP.   Is that not the case, and if not - why?

Reply:
@jsquire Yes it is possible to support that but not now, we could put this as a future plan.
I was trying to explain why we cannot do that *now*